src/Exceptions/Handler.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace Fuzz\ApiServer\Exceptions;
+
+
+use Exception;
+use Fuzz\HttpException\HttpException;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Database\QueryException;
+use Illuminate\Foundation\Exceptions\Handler as ExceptionHandler;
+use Illuminate\Http\JsonResponse;
+
+
+class Handler extends ExceptionHandler
+{
+	/**
+	 * A list of the exception types that should not be reported.
+	 *
+	 * @var array
+	 */
+	protected $dontReport = [
```

Comment:
Not sure that we should define exceptions to exclude from logging by default. This seems pretty application specific.